src/hotspot/cpu/riscv/macroAssembler_riscv.cpp

```diff
@@ -785,7 +785,7 @@ void MacroAssembler::emit_static_call_stub() {
 
   // Jump to the entry point of the c2i stub.
   int32_t offset = 0;
-  movptr(t1, 0, offset, t0); // lui + lui + slli + add
+  movptr2(t1, 0, offset, t0); // lui + lui + slli + add
```

Comment:
How about something like this?
```suggestion
  address placeholder = pc(); // correct value will be patched in later
  movptr2(t1, placeholder, offset, t0); // lui + lui + slli + add
```

Reply:
That seems OK to me. Then we can still use `movptr` here.
```
  address placeholder = pc(); // correct value will be patched in later
  movptr(t1, placeholder, offset, t0); // lui + lui + slli + add
```

Reply:
When looking at the disassembly it much easier to find uninitialized call stubs when we use 0.
Also if something would go wrong, it much nicer with a crash than a loop.
So for debuggability I prefer address 0.

Reply:
I'm inclined to leave this as is, because I agree with @robehn and because I think directly calling movptr2
when that's what one needs to have called (because one expected a particular sequence of instructions
to be generated) is better than indirectly calling it via the more generic movptr.  Anyone disagree?  I'll wait
another day for responses.

Reply:
I agree with a straight `0`.